content/posts/5-grzechow-juniora.mdx

```diff
@@ -0,0 +1,37 @@
+Nagle trafia się do projektu bez jakiegokolwiek doświadczenia, trzeba zdobyć teraz wiedzę domenową, dowiedzieć się kto jest kim itp.
+Dlatego by właśnie zminimalizować możliwe źródła stresu, dam ci parę rad na dobry początek.
+No to lecimy!
+
+## Proszenie o pomoc to nic złego!
+
+Niejednokrotnie na początku kariery będziesz napotykać taski np. oznaczone jako te „najłatwiejsze”, które przyblokują cię na dłuższy moment, bo w końcu nie posiadasz jeszcze odpowiedniej wiedzy domenowej bądź technicznej i jest to całkowicie normalne. Jednak nie popełniaj dość częstego błędu, którym jest pewnego rodzaju strach przed proszeniem o pomoc. Nie myśl, że twój zespół miałby cię z jakiegoś powodu wyśmiać, że nie umiesz zrobić X lub Y, bądź w ogóle zwolnić (tak, znam osoby co tak myślały). Każdy wie, że zostałeś zatrudniony jako junior i dobry zespół z wielką ochotą siądzie z tobą do taska, nakieruje na jego możliwe rozwiązanie i wytłumaczy parę kwestii, które cię dręczą. Kto pyta nie błądzi!
+
+## Skrajności w używaniu bibliotek.
+
+## Code Review to twoje najważniejsze źródełko wiedzy!
+
+Gdybym został zapytany za pomocą, czego nauczyłem się najwięcej odpowiedź byłaby tylko jedna: Code Review. Ten magiczny proces to idealna okazja do spokojnego przemyślenia i przyswojenia wiedzy na temat twoich lub czyichś błędów/propozycji do kodu, nie traktuj go jako kolejną przeszkodę, którą musisz pokonać by móc zamknąć taska. Nie zapomnij również, że twoje pomysły, czy komentarze nie są bezwartościowe tylko dlatego, że dopiero zaczynasz, pisz od razu i najwyżej rozpoczniesz ciekawa dyskusje, która nauczy czegoś ciebie i twoich rozmówców!
+
+## Pytania kieruj na publiczne kanały.
+
+Staraj się zadawać pytania na kanałach publicznych przeznaczonych dla określonej tematyki np. dla backendowców w projekcie. Jest to o wiele efektywniejsza taktyka, niż pisanie osoba po osobie prywatnie, gdyż każdy od razu widzi twoje pytanie i może akurat jest w temacie, przez co pomoc nadejdzie szybciej, lub nawet rozwinie się jakaś ciekawa dyskusja, na temat najlepszego rozwiązania.
+
+## Nie bądź tylko klepaczem kodu.
+
+Pewnie nie raz słyszałeś, jak kogoś nazywano „klepaczem kodu”. Kim jest taka osoba? W skrócie to po prostu programista, który niezbyt interesuje się rozwojem projektu, jest zazwyczaj mało pomysłowy na temat nowych dodatków do aplikacji, jego zmiany w kodzie często nie są jakkolwiek przemyślane, z code review wyciąga jedynie approve by móc zmerge'ować pull requesta i odpalić kolejnego taska, a jego celem jest po prostu przyjść do pracy wyklepać jak najszybciej zadania i do domu. Takie podejście bardzo szybko odbije się na twoim rozwoju. Staraj się poznawać projekt, w którym jesteś nie tylko od strony technicznej, ale równie biznesowej. Zadawaj pytania i staraj się samemu udzielać, gdy widzisz taką możliwość, nikt ci nic nie zrobi za to, że chciałeś dowiedzieć się czegoś więcej o własnym „dziecku”, które ty rozwijasz!
```

Comment:
Za długie trzecie zdanie, można by było podzielić je na kilka

Reply:
![image](https://user-images.githubusercontent.com/24841038/148175444-a60f56e7-e6ef-4897-93a5-2d67cb87d80f.jpeg)

Reply:
Przed nie przecinek (Staraj się poznawać projekt, w którym jesteś, nie tylko...)

Reply:
Ale równie**ż** biznesowej. 

Inaczej to zdanie nie ma sensu 😅

Reply:
Zadawaj pytania i staraj się samemu udzielać - co udzielać?

Reply:
które ty rozwijasz -> które rozwijasz